packages/shared/src/lib/image.ts

```diff
@@ -10,6 +10,8 @@ export const cloudinary = {
       layoutV1:
         'https://daily-now-res.cloudinary.com/image/upload/s--zkEyMCOC--/f_auto/v1705675789/public/Search_Gradient',
     },
+    betaTag:
+      'https://daily-now-res.cloudinary.com/image/upload/v1713111099/beta-tag_uiqqxx.svg',
```

Comment:
This URL is wrong, by default it should have the transformation enabled and not show the extension (that last bit basically shows you use the right URL)

Reply:
But its an SVG, we don't have transformations on SVGs right?

Reply:
Looking at other SVG URLs in file they are the same.

Reply:
They do have transformations afaik, should be provider atleast they give you that option.

Reply:
If I add it without extension and add transformation then we get white background and it is a webp image. Which is wrong.

<img width="146" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/f33ef37b-b016-4b47-beeb-0bcbc48f0c63">

Reply:
Quality is also worse then svg

Reply:
I see, ok let's leave it for now :)